substrate/client/network/sync/src/engine.rs

```diff
@@ -691,230 +696,222 @@ where
 		self.syncing_started = Some(Instant::now());
 
 		loop {
```

Comment:
Is the polling order important here? With the previous polling there was a fixed order, so maybe this select should be biased? https://docs.rs/tokio/latest/tokio/macro.select.html#fairness

Reply:
I don't think any select branch should be preferred when polling. That there was a specific order of polling before was more a limitation of serial polling, and it is not required when all futures are polled in parallel.